src/transformers/models/auto/tokenization_auto.py

```diff
@@ -137,6 +137,10 @@
             ("openai-gpt", ("OpenAIGPTTokenizer", "OpenAIGPTTokenizerFast" if is_tokenizers_available() else None)),
             ("gpt2", ("GPT2Tokenizer", "GPT2TokenizerFast" if is_tokenizers_available() else None)),
             ("gptj", ("GPT2Tokenizer", "GPT2TokenizerFast" if is_tokenizers_available() else None)),
+            (
+                "bloom",
+                ("BLOOMTokenizer", "BLOOMTokenizerFast" if is_tokenizers_available() else None),
+            ),
```

Comment:
This should all fit in one line
```suggestion
            ("bloom", ("BLOOMTokenizer", "BLOOMTokenizerFast" if is_tokenizers_available() else None)),
```